index.html

```
@@ -190,34 +152,82 @@ <h1 class="hdr-title">VoiceIsolate Pro <span class="hdr-ver">v20.0</span></h1>
+      <div class="diag-section">
+        <div class="diag-header">
+          <span class="diag-title">DIAGNOSTIC DASHBOARD</span>
+          <span class="diag-fps" id="diagFps">-- fps</span>
+        </div>
+
+        <!-- Row 1: A/B Waveform (full width) -->
+        <div class="card viz-card diag-ab">
+          <div class="viz-hdr">
+            <span>A/B Waveform Comparison</span>
+            <div class="viz-actions">
+              <button id="abOverlayBtn" class="btn btn-xs">Overlay</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)